source/_components/mopar.markdown

```diff
@@ -8,31 +8,40 @@ comments: false
 sharing: true
 footer: true
 logo: mopar.png
-ha_category: Car
+ha_category:
+  - Car
+  - Sensor
+  - Switch
+  - Lock
 ha_release: 0.53
 ha_iot_class: Cloud Polling
+redirect_from:
```

Comment:
Only the sensor link is needed, the rest can be removed.

Reply:
addressed